Cargo.toml

```diff
@@ -24,7 +24,9 @@ members = [
 adw = { version = "0.1", optional = true, package = "libadwaita" }
 async-trait = { version = "0.1", optional = true }
 fragile = "1.1"
+futures-channel = "0.3"
 futures-core = "0.3"
+futures-executor = "0.3"
```

Comment:
Shouldn't those also be dev-depencencies?

Reply:
Yes, pushed the change